tests/reports/test_codelist.py

```diff
@@ -289,7 +289,7 @@ def test_group_low_values(df, threshold):
     suppressed_count = result[result[code_column] == "Other"][count_column]
```

Comment:
Might be slightly nicer to return this as a numpy array and then index how it was originally
```suggestion
    suppressed_count = result.loc[result[code_column] == "Other", count_column].values
```

Reply:
Thank you. I admit, I don't really understand the change, but happy to trust you on pandas/numpy best practice!